src/nodejs/styles/colors.css

```diff
@@ -78,6 +78,7 @@
     --message-hover: #F8FBFE;
     --mention: #fed7aa;
     --mention-hover: #fb923c;
+    --mention-list-hover: #EDF2FA;
```

Comment:
May be better call it '--mention-list-item-selected'

Reply:
May be. I will rename it.